From: Ben Hutchings <ben@decadent.org.uk>
Date: Mon, 27 Nov 2017 19:36:06 +0000
Subject: cpumask: Avoid ABI change in 3.16.50
Forwarded: not-needed

Commit f36963c9d3f6 "cpumask_set_cpu_local_first => cpumask_local_spread,
lament" replaced the former function with the latter.  As these function
are exported and might be used by OOT modules, re-add
cpumask_set_cpu_local_first() as a wrapper around cpumask_local_spread().

---
--- a/lib/cpumask.c
+++ b/lib/cpumask.c
@@ -237,3 +237,10 @@ unsigned int cpumask_local_spread(unsign
 	BUG();
 }
 EXPORT_SYMBOL(cpumask_local_spread);
+
+int cpumask_set_cpu_local_first(int i, int numa_node, cpumask_t *dstp)
+{
+	cpumask_set_cpu(cpumask_local_spread(i, numa_node), dstp);
+	return 0;
+}
+EXPORT_SYMBOL(cpumask_set_cpu_local_first);
